lib/ruby_lsp/requests/hover.rb

```diff
@@ -28,18 +28,28 @@ class Hover < ExtensibleListener
         ],
         T::Array[T.class_of(Prism::Node)],
       )
+      GEMFILE_NAME = T.let(
+        begin
+          Bundler.with_original_env { Bundler.default_gemfile.basename.to_s }
+        rescue Bundler::GemfileNotFound
+          "Gemfile"
+        end,
+        String,
+      )
```

Comment:
This code is duplicated in `code_lens.rb`. Can we please move this into `utils.rb` and then just reference the constant in both `code_lens` and `hover`?